vault/identity_store_conflicts.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package vault
+
+import (
+	"context"
+	"errors"
+
+	log "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/helper/identity"
+)
+
+var errDuplicateIdentityName = errors.New("duplicate identity name")
```

Comment:
Looks like this is already declared in `identity_store_util`. I think it makes more sense for this to live here if this is where we're going to be making the determination that we have duplicates though so we can probably just remove it from there.

Reply:
Thanks @davidadeleon! Looks like vsCode didn't save my changes >.<